app/models/coach.rb

```diff
@@ -1,5 +1,7 @@
 class Coach < ApplicationRecord
 
+  has_many :players
+  has_many :teams
   has_many :players, through: :players_coaches
```

Comment:
There is no `has_many :player_coaches` to go `through` for this line. You should add that relationship and then this should all work. 


you will end up with both 

```ruby
has_many :player_coaches
has_many :players, through: :player_coaches
```